### PR TITLE
Do not invert colors of video and images

### PR DESCRIPTION
**Pull request by dcordero**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

Fix [#501](https://code.videolan.org/videolan/vlc-ios/issues/501)

### CODE REVIEW

File: Sources/MiniPlayer/AudioMiniPlayer.swift [hunk 1]
```diff
@@ -61,7 +61,7 @@ private extension AudioMiniPlayer {
 
         artworkImageView.clipsToBounds = true
         artworkImageView.layer.cornerRadius = 2
-
+
```
Comment (Mikanbu) [inline Sources/MiniPlayer/AudioMiniPlayer.swift]: Nit-pick: Extra unnecessary changes.

File: Sources/MiniPlayer/AudioMiniPlayer.swift [hunk 2]
```diff
@@ -180,6 +180,10 @@ private extension AudioMiniPlayer {
         artistLabel.text = metadata.artist
         if metadata.isAudioOnly {
             artworkImageView.image = metadata.artworkImage ?? UIImage(named: "no-artwork")
+            
+            if #available(iOS 11.0, *) {
+                artworkImageView.accessibilityIgnoresInvertColors = metadata.artworkImage != nil
```
Comment (Mikanbu) [inline Sources/MiniPlayer/AudioMiniPlayer.swift]: Shouldn't the `artworkImageView` always be set to true since in the case where `metadata.artworkImage` is nil, we are setting `artworkImageView.image` to a default artwork?

File: Sources/MiniPlayer/AudioMiniPlayer.swift (hunk 1 above)
Comment (dcordero) [inline Sources/MiniPlayer/AudioMiniPlayer.swift]:
```suggestion

```

File: Sources/MiniPlayer/AudioMiniPlayer.swift (hunk 2 above)
Comment (dcordero) [reply Sources/MiniPlayer/AudioMiniPlayer.swift]: I do not think so. The asset [no-artwork](https://github.com/videolan/vlc-ios/blob/d338742a878a6dda59e2fd7781097836802d955e/vlc-ios/Images.xcassets/MediaController/no-artwork.imageset/no-artwork%402x.png) is a plain image, inverting the colors should not be destructive on it.

Comment (Mikanbu) [reply Sources/MiniPlayer/AudioMiniPlayer.swift]: Indeed, I agree with you for the no-artwork!

Sadly I just thought of a special case. For the first version of the new UI we decided to use the AudioMiniPlayer as an VideoMiniPlayer as well! At least for now, this would means that we probably always need to set `accessibilityIgnoresInvertColors`.

**Comment by Mikanbu** [comment]
Rebased and merged with 740285e409f4cbd715bd075a7356d7538bc82af1.

I took the liberty to add the related ticket into the commit message.

Thank you for your contribution! 🎉
